packages/code-du-travail-frontend/src/modules/infographie/queries.ts

```diff
@@ -0,0 +1,153 @@
+import { elasticDocumentsIndex, elasticsearchClient } from "../../api/utils";
+import { SOURCES } from "@socialgouv/cdtn-utils";
+import {
+  DocumentElasticResult,
+  fetchDocument,
+  RelatedItem,
+  Source,
+} from "../documents";
+import { Infographic } from "./type";
+import { InfographicElasticDocument } from "@socialgouv/cdtn-types";
+import { LinkedContent } from "@socialgouv/cdtn-types/build/elastic/related-items";
+import { getRouteBySource } from "@socialgouv/cdtn-utils/src/sources";
+
+export const fetchInfographics = async <
+  K extends keyof InfographicElasticDocument,
+>(
+  fields: K[],
+  filters?: {
+    cdtnIds?: string[];
+  }
+): Promise<Pick<InfographicElasticDocument, K>[]> => {
+  const baseFilters: Array<any> = [
+    { term: { source: SOURCES.INFOGRAPHICS } },
+    { term: { isPublished: true } },
+  ];
+
+  if (filters?.cdtnIds) {
+    baseFilters.push({ terms: { cdtnId: filters.cdtnIds } });
+  }
+
+  const response = await elasticsearchClient.search<
+    Pick<InfographicElasticDocument, K>
+  >({
+    query: {
+      bool: {
+        filter: baseFilters,
+      },
+    },
+    size: 1000,
+    _source: fields,
+    index: elasticDocumentsIndex,
+  });
+  return response.hits.hits
+    .map(({ _source }) => _source)
+    .filter((model) => model !== undefined);
+};
+
+export const fetchInfographic = async <
+  K extends keyof InfographicElasticDocument,
+>(
+  slug: string,
+  fields: K[]
+): Promise<
+  DocumentElasticResult<Pick<InfographicElasticDocument, K>> | undefined
+> => {
+  return await fetchDocument<
+    InfographicElasticDocument,
+    keyof DocumentElasticResult<InfographicElasticDocument>
+  >(fields, {
+    query: {
+      bool: {
+        filter: [
+          { term: { source: SOURCES.INFOGRAPHICS } },
+          { term: { isPublished: true } },
+          { term: { slug } },
+        ],
+      },
+    },
+    size: 1,
+    _source: fields,
+    index: elasticDocumentsIndex,
+  });
+};
+
+export const format = ({
```

Comment:
```suggestion
export const formatInfographic = ({
```

je trouve ça plus simple pour retrouver avec l'autocomplete